superset-frontend/src/views/CRUD/data/savedquery/SavedQueryList.tsx

```diff
@@ -332,6 +331,15 @@ function SavedQueryList({
               icon: 'Copy',
               onClick: handleCopy,
             },
+            canExport
```

Comment:
It seems like canEdit/canExport/canDelete all have boolean values, is that correct? 

If so, would it be better to do like canEdit && the action as opposed to the ternary?

Reply:
Good catch, this is much cleaner!